ddsrouter_test/compose/test_cases/forwarding_routes/repeater/compose.yml

```diff
@@ -0,0 +1,105 @@
+# Test description:
+#   This test checks the repeater functionality of the DDS Router. A DDS Router is deployed in the cloud with a single
```

Comment:
Update description with routes information.

Reply:
I was asking for some word specific to forwarding routes feature, but that's fine.